rotkehlchen/tests/exchanges/test_bitfinex.py

```diff
@@ -867,18 +869,27 @@ def test_deserialize_asset_movement_deposit(mock_bitfinex):
         None,
     ]
     fee_asset = A_WBTC
-    expected_asset_movement = AssetMovement(
-        timestamp=Timestamp(1569348774),
+    expected_asset_movement = [AssetMovement(
+        timestamp=TimestampMS(1569348774000),
         location=Location.BITFINEX,
-        category=AssetMovementCategory.DEPOSIT,
-        address='DESTINATION_ADDRESS',
-        transaction_id='TRANSACTION_ID',
+        event_type=HistoryEventType.DEPOSIT,
         asset=fee_asset,
-        amount=FVal('0.26300954'),
-        fee_asset=fee_asset,
-        fee=Fee(FVal('0.00135')),
-        link=str(13105603),
-    )
+        balance=Balance(FVal('0.26300954')),
+        unique_id=str(13105603),
+        extra_data={
+            'movement_id': '13105603',
+            'address': 'DESTINATION_ADDRESS',
+            'transaction_id': 'TRANSACTION_ID',
+        },
+    ), AssetMovement(
+        timestamp=TimestampMS(1569348774000),
+        location=Location.BITFINEX,
+        event_type=HistoryEventType.DEPOSIT,
+        asset=fee_asset,
+        balance=Balance(FVal('0.00135')),
+        unique_id=str(13105603),
```

Comment:
here and in other places please fix. These are just string literals

```suggestion
        unique_id='13105603',
```